### PR TITLE
Recover accounts

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Also recover accounts.

## Changes

Recover accounts
Update UI in recovery page.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by orhoj** [comment]
The `Continue` button after having recovered is not placed correctly on the screen.

**Comment by orhoj** [comment]
After having recovered an account, there is not selected account in the account view. I can select the account, but it should be selected automatically after recovery.

**Comment by orhoj** [comment]
On failure the `Retry` button should be placed at the bottom like the `Continue` button.

File: packages/browser-wallet/src/background/recovery.ts [hunk 1]
```diff
@@ -1,23 +1,71 @@
...
+import { storedCurrentNetwork } from '@shared/storage/access';
+import { addCredential, addIdentity } from './update';
 
 // TODO: increase max empty
 // How many empty identityIndices are allowed before stopping
 const maxEmpty = 3;
 // Milliseconds to wait, if retrievalUrl is still pending
 const sleepInterval = 5000;
 
+function getAddressFrom(accountInfo: AccountInfo) {
```
Comment (orhoj) [inline packages/browser-wallet/src/background/recovery.ts:29]: You can do:
```
accountInfo.accountAddress
```

File: packages/browser-wallet/src/background/recovery.ts [hunk 2]
```diff
@@ -1,23 +1,71 @@
...
-import { Identity, CreationStatus, IdentityProvider } from '@shared/storage/types';
 import { BackgroundResponseStatus } from '@shared/utils/types';
-import { addIdentity } from './update';
+import { Identity, CreationStatus, IdentityProvider, WalletCredential } from '@shared/storage/types';
+import { storedCurrentNetwork } from '@shared/storage/access';
+import { addCredential, addIdentity } from './update';
 
 // TODO: increase max empty
 // How many empty identityIndices are allowed before stopping
 const maxEmpty = 3;
```
Comment (orhoj) [inline packages/browser-wallet/src/background/recovery.ts:25]: Update this to be 10, and remove the TODO.

**Review by orhoj — CHANGES_REQUESTED** [review]

File: packages/browser-wallet/src/background/recovery.ts (hunk 2 above)
Comment (shjortConcordium) [reply packages/browser-wallet/src/background/recovery.ts:25]: Because 10 is a good sample size?

Comment (orhoj) [reply packages/browser-wallet/src/background/recovery.ts]: Because I asked Lars to also use 10.

**Comment by orhoj** [comment]
The recovery screen CSS is messed up.

**Comment by shjortConcordium** [comment]
@orhoj I fixed the ui + updated the identity recovery to work with the new v1 version.

**Pull request merged**